spec/std/big/big_float_spec.cr

```diff
@@ -190,13 +191,16 @@ describe "BigFloat" do
   end
 
   describe "to_s" do
-    it { "0".to_big_f.to_s.should eq("0") }
+    it { "0".to_big_f.to_s.should eq("0.0") }
     it { "0.000001".to_big_f.to_s.should eq("0.000001") }
     it { "48600000".to_big_f.to_s.should eq("48600000") }
     it { "12345678.87654321".to_big_f.to_s.should eq("12345678.87654321") }
     it { "9.000000000000987".to_big_f.to_s.should eq("9.000000000000987") }
-    it { "12345678901234567".to_big_f.to_s.should eq("12345678901234567") }
-    it { "1234567890123456789".to_big_f.to_s.should eq("1234567890123456789") }
+    it { "12345678901234567".to_big_f.to_s.should eq("12345678901234567.0") }
+    it { "1234567890123456789".to_big_f.to_s.should eq("1234567890123456789.0") }
+
+    it { 3.to_big_f.to_s.should eq("3.0") }
+    it { (3.0).to_big_f.to_s.should eq("3.0") }
```

Comment:
I would write more test cases covering more cases. I was looking into this issue yesterday, and all the specs we have until now were covering `String` to `BigFloat`. The issue aims to cover `Float32 | Float64` to `BigFloat` and I wind up with trick responses like:

`it { 1.3.to_big_f.to_s.should eq("1.3") }`

```shell
Expected: "1.3"
got: "1.30000000000000004441"
```

`it { 0.000001.to_big_f.to_s.should eq("0.000001") }`

```shell
Expected: "0.000001"
got: "0.000000999999999999999954748"
```

For those cases, the numbers are Float64, and there is does not have an initializer on the BigFloat that covers `Floats` without passing the precision. For now, I extend more the test cases for floating numbers to see what need to be covered

Reply:
@rodrigopinto That's a relevant suggestion but it's unrelated to this PR's fix.

The problem you are describing is the result value of when you initialize a BigFloat with a Float. Not a part of this PR. This PR is good as is.

Reply:
I also saw there were some discussions about it on the issue. So, let's move ahead then.